### PR TITLE
Clarify beginning of user guide (loading data) 

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: brayns/common/utils/ImageUtils.cpp [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 #include <brayns/common/renderer/FrameBuffer.h>
 
+#ifdef BRAYNS_USE_FREEIMAGE
```
Comment (rolandjitsu) [inline brayns/common/utils/ImageUtils.cpp]: This is not user guide 😄

Comment (rdumusc) [reply brayns/common/utils/ImageUtils.cpp]: That's why this commit is called "Fix build without FreeImage" ;-)

Comment (rolandjitsu) [reply brayns/common/utils/ImageUtils.cpp]: I was looking at the PR name 😄

Comment (rdumusc) [reply brayns/common/utils/ImageUtils.cpp]: I'm not going to squash the commits, and I don't want to open a separate PR for such a small fix...

**Pull request merged**